src/frontend/src/utils/gantt.utils.ts

```diff
@@ -40,7 +41,7 @@ export const filterGanttProjects = (projects: Project[], ganttFilters: GanttFilt
     return project.status.toString() === ganttFilters.status;
   };
   const teamCheck = (project: Project) => {
-    return project.team === undefined ? decodedTeam === NO_TEAM : project.team?.teamName === decodedTeam;
+    return getProjectTeamsName(project) === decodedTeam;
```

Comment:
this should check if the TeamsName contains the decodedTeam, that way if you filter on any of the teams on a project, it'll show up (this is why we need better seed data, this is kinda hard to test locally)

Reply:
oh yeah ig ur right